{
  "artifacts": [],
  "command": "my_main",
  "experiment": {
    "base_dir": "C:\\Users\\hsk\\Desktop\\python\\pymarl",
    "dependencies": [
      "numpy==1.23.1",
      "PyYAML==6.0.2",
      "sacred==0.8.7",
      "torch==1.13.1+cu117"
    ],
    "mainfile": "main.py",
    "name": "pymarl",
    "repositories": [
      {
        "commit": "b3bac7b3ce67136c02baa0bb887d872793fae327",
        "dirty": true,
        "url": "https://github.com/Kai05012002/testpymarl"
      },
      {
        "commit": "b3bac7b3ce67136c02baa0bb887d872793fae327",
        "dirty": true,
        "url": "https://github.com/Kai05012002/testpymarl"
      },
      {
        "commit": "b3bac7b3ce67136c02baa0bb887d872793fae327",
        "dirty": true,
        "url": "https://github.com/Kai05012002/testpymarl"
      }
    ],
    "sources": [
      [
        "main.py",
        "_sources\\main_319c15f2fb11d58696da491439d5dbb5.py"
      ],
      [
        "run.py",
        "_sources\\run_3d13dbdeea410bcf5bb87beb4266819c.py"
      ],
      [
        "utils\\logging.py",
        "_sources\\logging_1c1c5c83be7ac7b7b69888b3bcb127e3.py"
      ]
    ]
  },
  "fail_trace": [
    "Traceback (most recent call last):\n",
    "  File \"c:\\users\\hsk\\desktop\\python\\smac\\smac\\env\\starcraft2\\starcraft2.py\", line 420, in step\n    self._controller.actions(req_actions)\n",
    "  File \"C:\\Users\\hsk\\anaconda3\\envs\\SMAC0104\\lib\\site-packages\\pysc2\\lib\\remote_controller.py\", line 81, in _skip_status\n    return func(self, *args, **kwargs)\n",
    "  File \"C:\\Users\\hsk\\anaconda3\\envs\\SMAC0104\\lib\\site-packages\\pysc2\\lib\\remote_controller.py\", line 92, in _valid_status\n    raise protocol.ProtocolError(\n",
    "pysc2.lib.protocol.ProtocolError: `actions` called while in state: Status.ended, valid: (Status.in_game)\n",
    "\nDuring handling of the above exception, another exception occurred:\n\n",
    "Traceback (most recent call last):\n",
    "  File \"C:\\Users\\hsk\\anaconda3\\envs\\SMAC0104\\lib\\site-packages\\sacred\\config\\captured_function.py\", line 42, in captured_function\n    result = wrapped(*args, **kwargs)\n",
    "  File \"C:\\Users\\hsk\\Desktop\\python\\pymarl\\main.py\", line 36, in my_main\n    run(_run, config, _log)\n",
    "  File \"C:\\Users\\hsk\\Desktop\\python\\pymarl\\run.py\", line 51, in run\n    run_sequential(args=args, logger=logger)\n",
    "  File \"C:\\Users\\hsk\\Desktop\\python\\pymarl\\run.py\", line 210, in run_sequential\n    episode_batch = runner.run(test_mode=False)\n",
    "  File \"C:\\Users\\hsk\\Desktop\\python\\pymarl\\runners\\episode_runner.py\", line 73, in run\n    reward, terminated, env_info = self.env.step(actions[0])\n",
    "  File \"c:\\users\\hsk\\desktop\\python\\smac\\smac\\env\\starcraft2\\starcraft2.py\", line 426, in step\n    self.full_restart()\n",
    "  File \"c:\\users\\hsk\\desktop\\python\\smac\\smac\\env\\starcraft2\\starcraft2.py\", line 393, in full_restart\n    self._launch()\n",
    "  File \"c:\\users\\hsk\\desktop\\python\\smac\\smac\\env\\starcraft2\\starcraft2.py\", line 300, in _launch\n    self._sc2_proc = self._run_config.start(window_size=self.window_size, want_rgb=False)\n",
    "  File \"C:\\Users\\hsk\\anaconda3\\envs\\SMAC0104\\lib\\site-packages\\pysc2\\run_configs\\platforms.py\", line 83, in start\n    return sc_process.StarcraftProcess(\n",
    "  File \"C:\\Users\\hsk\\anaconda3\\envs\\SMAC0104\\lib\\site-packages\\pysc2\\lib\\sc_process.py\", line 137, in __init__\n    self._controller = remote_controller.RemoteController(\n",
    "  File \"C:\\Users\\hsk\\anaconda3\\envs\\SMAC0104\\lib\\site-packages\\pysc2\\lib\\remote_controller.py\", line 142, in __init__\n    sock = self._connect(host, port, proc, timeout_seconds)\n",
    "  File \"C:\\Users\\hsk\\anaconda3\\envs\\SMAC0104\\lib\\site-packages\\pysc2\\lib\\stopwatch.py\", line 205, in _stopwatch\n    return func(*args, **kwargs)\n",
    "  File \"C:\\Users\\hsk\\anaconda3\\envs\\SMAC0104\\lib\\site-packages\\pysc2\\lib\\remote_controller.py\", line 176, in _connect\n    raise ConnectError(\"Failed to connect to the SC2 websocket. Is it up?\")\n",
    "pysc2.lib.remote_controller.ConnectError: Failed to connect to the SC2 websocket. Is it up?\n"
  ],
  "heartbeat": "2025-02-14T14:25:30.559757",
  "host": {
    "ENV": {},
    "cpu": "Intel(R) Core(TM) i5-14500",
    "gpus": {
      "driver_version": "566.03",
      "gpus": [
        {
          "model": "NVIDIA GeForce RTX 4070 Ti SUPER",
          "persistence_mode": false,
          "total_memory": 16376
        }
      ]
    },
    "hostname": "DESKTOP-68E0S8K",
    "os": [
      "Windows",
      "Windows-10-10.0.19045-SP0"
    ],
    "python_version": "3.10.14"
  },
  "meta": {
    "command": "my_main",
    "config_updates": {
      "checkpoint_path": "C:/Users/hsk/Desktop/python/pymarl/results/models/enhanced_gat_1m_dqn__2025-02-14_09-19-03",
      "env_args": {
        "map_name": "1m_vs_1m",
        "seed": 123
      },
      "load_step": 1760713,
      "local_results_path": "./results",
      "save_model": true,
      "save_model_interval": 20000,
      "t_max": 4000000,
      "use_tensorboard": true
    },
    "named_configs": [],
    "options": {
      "--beat-interval": null,
      "--capture": null,
      "--comment": null,
      "--debug": false,
      "--enforce_clean": false,
      "--file_storage": null,
      "--force": false,
      "--help": false,
      "--id": null,
      "--loglevel": null,
      "--mongo_db": null,
      "--name": null,
      "--pdb": false,
      "--print-config": false,
      "--priority": null,
      "--queue": false,
      "--s3": null,
      "--sql": null,
      "--tiny_db": null,
      "--unobserved": false,
      "COMMAND": null,
      "UPDATE": [
        "env_args.map_name=1m_vs_1m",
        "env_args.seed=123",
        "use_tensorboard=True",
        "save_model=True",
        "save_model_interval=20000",
        "local_results_path=./results",
        "t_max=4000000",
        "checkpoint_path=C:/Users/hsk/Desktop/python/pymarl/results/models/enhanced_gat_1m_dqn__2025-02-14_09-19-03",
        "load_step=1760713"
      ],
      "help": false,
      "with": true
    }
  },
  "resources": [],
  "result": null,
  "start_time": "2025-02-14T13:31:22.176754",
  "status": "FAILED",
  "stop_time": "2025-02-14T14:25:30.587757"
}